packages/suite/src/components/suite/layouts/SuiteLayout/Sidebar/NavigationItem.tsx

```diff
@@ -109,10 +124,18 @@ export const NavigationItem = ({
             className={className}
             tabIndex={0}
             $elevation={elevation}
+            $isRounded={isRounded}
+            $typographyStyle={typographyStyle}
         >
             <Icon name={icon} size={iconSize} color={theme.iconSubdued} pointerEvents="none" />
-            <Translation id={nameId} values={values} />
-            {itemsCount && <Count>{itemsCount}</Count>}
+            <Paragraph variant="tertiary" typographyStyle={typographyStyle}>
```

Comment:
This wasn't grey before, right? I think the tertiary only applies to the count.

Reply:
True, good catch